lib/modules/dosomething/dosomething_reportback/dosomething_reportback.admin.inc

```diff
@@ -290,13 +289,19 @@ function dosomething_reportback_files_form_submit($form, &$form_state) {
       $rogue_item_id = dosomething_rogue_get_by_file_id($fid);
 
       // If there is a rogue_item_id in the dosomething_rogue_reportbacks table, send update to Rogue.
-      if (!is_null($rogue_item_id[0]->rogue_item_id)) {
+      if ($rogue_item_id[0]->rogue_item_id) {
+        if ($values['status'] === 'promoted' || $values['status'] === 'approved' || $values['status'] === 'excluded') {
+          $values['status'] = 'accepted';
```

Comment:
I'm not sure if naming this will cause issues with how it's saved or something, might be nice to save it as `rogue_value` or something so there isn't a collision